### PR TITLE
PTC glTF extension

### PR DESCRIPTION
**Pull request by PaulJPremakumar**
- Added the ReadMe
- Added the schema file

- Validated schema file using : https://www.jsonschemavalidator.net/

### CODE REVIEW

File: extensions/2.0/Vendor/PTC_onshape_metadata/schema/gltf.PTC_onshape_metadata.schema.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "id": {
+            "anyOf":[
+                {
+                    "type": "array",
+                    "minItems": 1,
+                    "items": {
+                        "type": "string"
+                    }
```
Comment (evannowak) [inline extensions/2.0/Vendor/PTC_onshape_metadata/schema/gltf.PTC_onshape_metadata.schema.json]: Should this have a minLength: 1?

File: extensions/2.0/Vendor/PTC_onshape_metadata/schema/gltf.PTC_onshape_metadata.schema.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                    "type": "string",
+                    "minLength": 1
+                }
+            ],
+            "description": "An identifier for this object."
+        },
+        "hidden": {
+            "type": "boolean",
+            "description": "If true, the object should not be rendered."
+        },
```
Comment (evannowak) [inline extensions/2.0/Vendor/PTC_onshape_metadata/schema/gltf.PTC_onshape_metadata.schema.json]: How about: "If true, the object and its descendants should not be rendered."

**Review by evannowak — CHANGES_REQUESTED** [review]
Looks good. A couple minor comments attached.

File: extensions/2.0/Vendor/PTC_onshape_metadata/schema/gltf.PTC_onshape_metadata.schema.json (hunk 1 above)
Comment (PaulJPremakumar) [reply extensions/2.0/Vendor/PTC_onshape_metadata/schema/gltf.PTC_onshape_metadata.schema.json]: Good catch. Added the validation.

File: extensions/2.0/Vendor/PTC_onshape_metadata/schema/gltf.PTC_onshape_metadata.schema.json (hunk 2 above)
Comment (PaulJPremakumar) [reply extensions/2.0/Vendor/PTC_onshape_metadata/schema/gltf.PTC_onshape_metadata.schema.json]: Updated.

**Comment by PaulJPremakumar** [comment]
Addressed changes.

**Comment by PaulJPremakumar** [comment]
Addressed changes.

**Pull request merged**